packages/node_modules/@webex/webex-core/src/lib/services/service-url.js

```diff
@@ -83,7 +97,9 @@ const ServiceUrl = AmpState.extend({
     const {hostname} = Url.parse(url);
     const foundHost = this.hosts.find((hostObj) => hostObj.host === hostname);
 
-    this.hosts.splice(this.hosts.indexOf(foundHost), 1);
+    if (foundHost) {
+      foundHost.failed = true;
```

Comment:
Would it be better to rename this function `failHost` otherwise, you have a mental overhead added to understand that failed is the same as removed?

Reply:
Yep, implementing this.